lighthouse-cli/sentry-prompt.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+import {Configstore, inquirer} from './shim-modules';
+
+const log = require('lighthouse-logger');
+
+const MAXIMUM_WAIT_TIME = 20 * 1000;
+// clang-format off
+const MESSAGE =
+  `${log.reset}Lighthouse is requesting permission to anonymously report back runtime exceptions.\n  ` +
+  `${log.reset}This can include data such as the test URL, its subresources, your OS, Chrome version, and Lighthouse version.\n  ` +
+  `May ${log.green}Lighthouse${log.reset} ${log.bold}report this data to aid in improving the tool?`;
+// clang-format on
+
+async function prompt() {
+  if (!process.stdout.isTTY || process.env.CI) {
+    // Default non-interactive sessions to false
```

Comment:
wdyt about defaulting CI to true?

Reply:
I'm split, I'm definitely against in *our* CI so an opt-out env variable seems good.

It'd be nice to gather errors from folks running Lighthouse in CI, but silently opting them in seems bad. Can always use `--enable-error-reporting` to skip this part entirely and run with error reporting so I'd vote no to default CI true.

Reply:
I do like the idea of enabling this for our smoke tests on master though  👍  followup work

Reply:
ok sg